navigator_engine/config.py

```diff
@@ -10,16 +10,21 @@ class Config(object):
     SQLALCHEMY_DATABASE_URI = 'postgresql://postgres:1234567890@db/navigator_engine'
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     LOGGING_LEVEL = logging.INFO
+    DASH_REROUTE_PREFIX = '/engine'
 
 
 class Testing(Config):
     TESTING = True
     SQLALCHEMY_DATABASE_URI = f'sqlite+pysqlite:////{base_directory}/../testing.db'
+    TEST_DATA_SPREADSHEET = f'{base_directory}/tests/test_data/Estimates Test Data.xlsx'
+    TEST_DATA_GRAPH_FOLDER = f'{base_directory}/tests/test_data/test_graphs/'
```

Comment:
Suggest `DECISION_GRAPH_FOLDER`

Reply:
Context... Trying to avoid calling the graph "data" because we are dealing with a tonne of "data" elsewhere in the engine. Keen to make the distinction between"data" and the "decision graph".